src/SerilogWeb.Classic/SerilogWebClassicLoggerConfigurationExtensions.cs

```diff
@@ -0,0 +1,186 @@
+            this LoggerEnrichmentConfiguration enrichmentConfiguration,
+            bool checkForHttpProxies = true)
+        {
+            if (enrichmentConfiguration == null) throw new ArgumentNullException(nameof(enrichmentConfiguration));
+            return enrichmentConfiguration.With(new HttpRequestClientHostIPEnricher(checkForHttpProxies));
+        }
+
+        /// <summary>
+        /// Enrich log events with the Client Host Name.
+        /// </summary>
+        /// <param name="enrichmentConfiguration">Logger enrichment configuration.</param>
+        /// <returns>Configuration object allowing method chaining.</returns>
+        public static LoggerConfiguration WithHttpRequestClientHostName(
+            this LoggerEnrichmentConfiguration enrichmentConfiguration)
+        {
+            if (enrichmentConfiguration == null) throw new ArgumentNullException(nameof(enrichmentConfiguration));
+            return enrichmentConfiguration.With<HttpRequestClientHostNameEnricher>();
+        }
+
+        /// <summary>
+        /// Enrich log events with a HttpRequestId GUID.
+        /// </summary>
+        /// <param name="enrichmentConfiguration">Logger enrichment configuration.</param>
+        /// <returns>Configuration object allowing method chaining.</returns>
+        public static LoggerConfiguration WithHttpRequestId(
+            this LoggerEnrichmentConfiguration enrichmentConfiguration)
+        {
+            if (enrichmentConfiguration == null) throw new ArgumentNullException(nameof(enrichmentConfiguration));
+            return enrichmentConfiguration.With<HttpRequestIdEnricher>();
+        }
+
+        /// <summary>
+        /// Enrich log events with a HttpRequestNumber unique within the current
+        /// logging session.
+        /// </summary>
+        /// <param name="enrichmentConfiguration">Logger enrichment configuration.</param>
+        /// <returns>Configuration object allowing method chaining.</returns>
+        public static LoggerConfiguration WithHttpRequestNumber(
+            this LoggerEnrichmentConfiguration enrichmentConfiguration)
+        {
+            if (enrichmentConfiguration == null) throw new ArgumentNullException(nameof(enrichmentConfiguration));
+            return enrichmentConfiguration.With<HttpRequestNumberEnricher>();
+        }
+
+        /// <summary>
+        /// Enrich log events with the Url of the Request.
+        /// </summary>
+        /// <param name="enrichmentConfiguration">Logger enrichment configuration.</param>
+        /// <returns>Configuration object allowing method chaining.</returns>
+        public static LoggerConfiguration WithHttpRequestUrl(
+            this LoggerEnrichmentConfiguration enrichmentConfiguration)
+        {
+            if (enrichmentConfiguration == null) throw new ArgumentNullException(nameof(enrichmentConfiguration));       
+            return enrichmentConfiguration.With< HttpRequestUrlEnricher>();
```

Comment:
nitpicking : there is a whitespace between `With<` and `HttpRequestUrlEnricher` :)